pkgs/servers/web-apps/discourse/update.py

```diff
@@ -35,6 +38,15 @@ def tags(self) -> Iterable[str]:
         versions.sort(key=lambda x: LooseVersion(x.replace('v', '')), reverse=True)
         return versions
 
+    @property
+    def latest_commit_sha(self) -> str:
+        if self._latest_commit_sha is None:
+            r = requests.get(f'https://api.github.com/repos/{self.owner}/{self.repo}/commits?per_page=1')
```

Comment:
Does this use a token?

Reply:
No, you can make these API requests without one. Having one probably lifts the rate limit, but it's at 60 requests without a token, which is plenty.